CHANGELOG.md

```diff
@@ -2238,7 +2238,7 @@ in d.ts.
 ### Bug Fixes
 
 - **micro-perf:** rename immediate to queue scheduler ([fe56b28](https://github.com/ReactiveX/RxJS/commit/fe56b28)), closes [#1040](https://github.com/ReactiveX/RxJS/issues/1040)
-- **micro-perf:** use the correnct scheduler on current-thread tests ([3dff5eb](https://github.com/ReactiveX/RxJS/commit/3dff5eb))
+- **micro-perf:** use the current scheduler on current-thread tests ([3dff5eb](https://github.com/ReactiveX/RxJS/commit/3dff5eb))
```

Comment:
I'm still not sure. I wasn't sure when Google Sheets offered a suggestion, and I wasn't sure after staring at the code or the PR for a while.